src/desktop.js

```diff
@@ -130,8 +130,8 @@ export default class Desktop extends EventEmitter {
      * @type {Search|null}
      * @readonly
      */
-    this.search = core.config('search.enabled') ? new Search(core) : null;
-
+    //this.search = core.config('search.enabled') ? new Search(core) : null;
```

Comment:
Don't include commented code.

Reply:
> Don't include commented code.

You were supposed to tell me where I changed the wrong code and I will go and correct it